Lib/test/test_os.py

```diff
@@ -3958,8 +3958,8 @@ def test_bytes_like(self):
             self.assertIs(type(entry.name), bytes)
             self.assertIs(type(entry.path), bytes)
 
-    @unittest.skipUnless(os.listdir in os.supports_fd,
-                         'fd support for listdir required for this test.')
+    @unittest.skipUnless(os.scandir in os.supports_fd,
```

Comment:
Could you put a TODO comment here for changing it back to `listdir` once scandir has supports_fd?

Reply:
Yes, I should do this.